fresnel_plane/materials/scripts/fresnel_fp.glsl

```diff
@@ -0,0 +1,41 @@
+#version 150
```

Comment:
Are we able to use versoin 150 with Gazebo? Does it matter?

Reply:
yeah version 150 works but probably not compatible with the waves ocean model